src/main/java/racing/domain/Simulator.java

```diff
@@ -15,9 +16,15 @@ public void create(String names) {
         }
     }
 
+    public void move(Car car, int random) {
```

Comment:
`Simulator` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private void move(Car car, int random) {
```

Reply:
`move`와 `run` 둘 다 Test에만 사용하고 있어서, `private`로 변경하고, 외부/내부적으로 사용하는 `simulate` 함수를 테스트하는 것으로 대체하였습니다.